testbed/compositions/rfc_k8s.toml

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name    = "rfcBBL103A"
+author  = "adlrocha"
+
+[global]
+plan    = "testbed"
+case    = "ipfs-transfer"
+builder = "docker:generic"
+runner  = "cluster:k8s"
+
+total_instances = 6
+
+[global.build_config]
+  push_registry=true
+  go_proxy_mode="remote"
+  go_proxy_url="http://localhost:8081"
+  registry_type="aws"
+
+[[groups]]
+id = "nodes"
+instances = { count = 6 }
+
```

Comment:
I've removed the `dependencies` from the other composition file, since they don't compile - I guess they are outdated.